packages/suite/src/storage/migrations/index.ts

```diff
@@ -147,4 +148,47 @@ export const migrate = async (
             cursor = await cursor.continue();
         }
     }
+
+    if (oldVersion < 21) {
+        // do the same thing as in blockchain-link's transformTransaction
```

Comment:
next time, we need to implement additional logic to trigger refetch of all transactions. this is pain.